packages/twenty-server/src/modules/workspace-member/repositories/workspace-member.repository.ts

```diff
@@ -28,6 +28,20 @@ export class WorkspaceMemberRepository {
     return result;
   }
 
+  public async find(workspaceMemberId: string, workspaceId: string) {
```

Comment:
nit: I would have named it findById to align with typeorm
Anyway, we are moving to the new ORM very soon so I think it's fine!